next.config.js

```diff
@@ -9,6 +9,8 @@ const ContentSecurityPolicy = `
       : ""
   };
   child-src 'self';
+  frame-src 'self' https://getsession.org https://optf.ngo https://oxen.io https://staging.getsession.org https://staging.oxen.io;
```

Comment:
we only `need frame-src 'self';` and `frame-ancestors 'self';`  tbh

Reply:
I had suggested we do it across all sites in case we wanted to reference different webpages from our different products in an iframe. This has become redundant since we have removed the iframes and just linked to the relevant web page.